spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -0,0 +1,7 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_mailer
+class UserMailerPreview < ActionMailer::Preview
+  def friend_request_email
+    friendship = FactoryBot.build(:friendship)
+    UserMailer.friend_request_email(friendship)
```

Comment:
отделяй отличающиеся вещи друг от друга так проще читать, тут в одном блоке определяешь переменную, в последнем вызываешь мейлер
```suggestion
    friendship = FactoryBot.build(:friendship)

    UserMailer.friend_request_email(friendship)
```